packages/core/src/storage/types.ts

```diff
@@ -159,3 +160,37 @@ export interface ThreadSortOptions {
 export type ThreadOrderBy = 'createdAt' | 'updatedAt';
 
 export type ThreadSortDirection = 'ASC' | 'DESC';
+
+export interface AISpanRecord {
+  traceId: string;
+  spanId: string;
+  parentSpanId: string | null;
+  name: string;
+  scope: Record<string, any> | null;
+  spanType: AISpanType;
+  attributes: Record<string, any> | null;
+  metadata: Record<string, any> | null;
+  events: Record<string, any> | null;
+  links: Record<string, any> | null;
+  other: string | null;
+  startAt: Date;
+  endAt: Date | null;
+  createdAt: Date;
+  updatedAt: Date | null;
+  input: Record<string, any> | null;
+  output: Record<string, any> | null;
+  error: Record<string, any> | null;
```

Comment:
lets make input, output, error & links just `any` type.

Reply:
or something json blob compatible... 

they can be a `string` or a `number` or a `list` for example, which wont fit into `Record<string, any>`